components/app-configs/mlde/workspace_config/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: <replace me>
+
+resources:
+- namespace.yaml
+- worker-service-account.yaml
+- scc-kludge.yaml
+
+secretGenerator:
+- name: azure-service-principal
+  files:
+  - kubernetes.service-principal.json
+
+generatorOptions:
+  disableNameSuffixHash: true
```

Comment:
I am new to kustomize, unsure if there is a better way to not append a UID to the generated secret. I want the secret to be the same on every namespace to simplify deployments of new workspaces.